### ISSUE TITLE
Duplicated output on rmPreProcessing?

### ISSUE THREAD

**Issue by wleoncio** [SystemsBiologist/CellMigRation#22]
# Summary

The output from the README example seems to yield duplicate output.

# Script

```R
library(CellMigRation) 
     
data(TrajectoryDataset) 
df <- CellMig(TrajectoryDataset[1:100, ]) 
preproc <- rmPreProcessing(df, PixelSize = 1.24, TimeInterval = 100)     
```

# Observed output
![image](https://user-images.githubusercontent.com/8234768/87016524-1e5ca300-c1cf-11ea-8567-3cc183a3c79d.png)

# Expected output

Only one line starting with "This dataset contains".

# Proposed solutions

1. Delete [line 3718](https://github.com/ocbe-uio/CellMigRation/blob/master/R/all_functions.R#L3718) or [line 3735](https://github.com/ocbe-uio/CellMigRation/blob/master/R/all_functions.R#L3735)
2. Improve messages so output is clearer to user (e.g. add "also contains" to [L3735](https://github.com/ocbe-uio/CellMigRation/blob/master/R/all_functions.R#L3735))

**Comment by SystemsBiologist** [comment]
Hi Waldir,
The TimeInterval is 10 not 100.
 In this dataset the duplicated output is meaningless but if we take the output of the tracking from Damiano's part and run the preprocessing then the output will not be duplicated because first the function will exclude the cells with less than three steps but of course if you think it is not important to show the total number of cells in the dataset we can remove the first line and keep the second.

**Comment by wleoncio** [comment]
So maybe the first line should say "This dataset contains: 1 Cell in total"? I think this differentiates that content from the one on the second line well enough.

**Comment by SystemsBiologist** [comment]
I totally agree.

**Comment by wleoncio** [comment]
Solved on 4e430c18f42b3e6440b14f862effd9a99644b1be.
